eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Rewards.hs

```diff
@@ -754,6 +754,7 @@ aggDeltaRewardEvents :: [ChainEvent C] -> Map (Credential 'Staking (Crypto C)) (
 aggDeltaRewardEvents events = foldl' accum Map.empty events
   where
     accum ans (TickEvent (Tick.RupdEvent (RupdEvent _ m))) = Map.unionWith Set.union m ans
+    accum ans (TickEvent (Tick.NewEpochEvent (DeltaRewardEvent (RupdEvent _ m)))) = Map.unionWith Set.union m ans
```

Comment:
Do I understand this event correctly? UNfortunately there is no description about it.

Reply:
Refering to my commit message 
```
This is an issue that the generators are not strong enough to discover. The DeltaRewardEvent appears in rare 
cases when there are no blocks after the 80% of an epoch. For these cases it has to be taken into account while
 aggregating incremental rewards
```